mri/reconstruct/fourier.py

```diff
@@ -208,3 +208,139 @@ def adj_op(self, x):
         """
         self.plan.f = x
         return np.copy(self.plan.adjoint()) / np.sqrt(self.plan.M)
+
+
+class Stacked3D:
+    """"  3-D non uniform Fast Fourrier Transform class,
```

Comment:
check typos: Fourrier -> Fourier

Reply:
Ahh, thank you @philouc . I will get this fixed in the next commit.